apollo-backend/src/test/java/io/logz/apollo/BlockerTest.java

```diff
@@ -41,7 +38,13 @@ public void testEverythingBlocker() throws Exception {
 
         BlockerDefinition blocker = createAndSubmitBlocker(apolloTestAdminClient, "unconditional", "{}", null, null);
 
-        assertThatThrownBy(() -> ModelsGenerator.createAndSubmitDeployment(apolloTestClient, environment, service, deployableVersion)).isInstanceOf(ApolloBlockedException.class);
+        try {
+            ModelsGenerator.createAndSubmitDeployment(apolloTestClient, environment, service, deployableVersion);
+            // If we got to this assertion, then we have a blocker issue:
+            assertThat(true).isEqualTo(false);
```

Comment:
This type of issues should always indicate that there is something wrong with the design :)
We must pass the exception as an actual parameter to the responseObject, and assert on that